src/System.Private.Xml/src/System/Xml/BinaryXml/SqlUtils.cs

```diff
@@ -606,10 +606,10 @@ public static DateTime SqlDateTimeToDateTime(int dateticks, uint timeticks)
         }
 
         // Number of (100ns) ticks per time unit
-        private static readonly double s_SQLTicksPerMillisecond = 0.3;
-        public static readonly int SQLTicksPerSecond = 300;
-        public static readonly int SQLTicksPerMinute = SQLTicksPerSecond * 60;
-        public static readonly int SQLTicksPerHour = SQLTicksPerMinute * 60;
+        private const double s_SQLTicksPerMillisecond = 0.3;
+        public const int SQLTicksPerSecond = 300;
+        public const int SQLTicksPerMinute = SQLTicksPerSecond * 60;
+        public const int SQLTicksPerHour = SQLTicksPerMinute * 60;
```

Comment:
Can you make these `internal const` instead?

Reply:
Sure.